### PR TITLE
Bug Fixes for Player

### PR DESCRIPTION
**Pull request by 0xcadams**
## Description

Fixes types too strict on `<Player />`, background play/pause click handler not working without a poster element, theme overrides without a global theme, and styling for the container to take up the parent width and maintain aspect ratio.

### CODE REVIEW

File: examples/next/src/pages/player.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+function toStringValues(obj: Record<string, any>) {
+  const strObj: Record<string, string> = {};
+  for (const [key, value] of Object.entries(obj)) {
+    strObj[key] = value.toString();
+  }
+  return strObj;
+}
+
+const PlayerPage: NextPage = () => {
```
Comment (victorges) [inline examples/next/src/pages/player.tsx:17]: Is this reusable for the lvpr.tv repo somehow? Seems like it's the exact same code right 👀

File: packages/react/src/components/media/Player.tsx [hunk 2]
```diff
@@ -69,26 +69,7 @@ export type PlayerProps = {
 
   /** The refetch interval for the playback info hook (used with `playbackId` to query until there is a valid playback URL) */
   refetchPlaybackInfoInterval?: number;
-} & (
-  | {
-      src: string | string[] | null | undefined;
```
Comment (victorges) [inline packages/react/src/components/media/Player.tsx:74]: Ah thanks! I did have issues to get typescript to like what I was setting here. Wdyt of having only the requirement of having an src or playbackId? Just so it complains if you don't set anything. something like
```ts
& ({src: string|...} | {playbackId: string|...})
```

**Review by victorges — APPROVED** [review]
Nice!

File: packages/react/src/components/media/Player.tsx (hunk 2 above)
Comment (0xcadams) [reply packages/react/src/components/media/Player.tsx:74]: Addressed in https://github.com/livepeer/livepeer.js/commit/703ff7552e79a6e9994439cb05a1446c6e5037f4

**Pull request merged**